prototype/water-extent-map.ipynb

```diff
@@ -0,0 +1,242 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# HyP3 prototype water extent map product\n",
+    "\n",
+    "This notebook mimics the workflow a HyP3 Water Map plugin will follow to create an On Demand surface water extent map. It is intended demonstrate *how* HyP3 goes from an On Demand request to a product as well as provide a platform for testing/tuning/refinement ot the water extent map product.\n",
+    "\n",
+    "Importantly, step 3 will become the plugin's primary inteface and the options provided to the `make_water_map` function will be the ones provided to users (or a subset thereof). Steps 1 and 2 are required data marshelling steps that will be handled transparently inside of the container.\n",
+    "\n",
+    "If you'd like an overview of the water extent mapping process, run the following cell to get the `make_water_map` documentation, otherwise, skip to step 1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from asf_tools.water_map import make_water_map\n",
+    "\n",
+    "?make_water_map"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 0. Current limitations\n",
+    "\n",
+    "* RTC products leverage the [Copernicus GLO-30 Public DEM](https://hyp3-docs.asf.alaska.edu/dems) and have *global* coverage, but we currently need to **manually create HAND data** for each Area of Interest (AIO). We are working on generating global HAND data and will update this notebook when it becomes available. Until then, if you want to look at an area outside the bounds of the HAND source file used in this notebook (see step 2), you can [ask us](https://gitter.im/ASFHyP3/community) or use [hand.py](../asf_tools/hand.py) to generate HAND data over your AIO. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Generate input dual-pol RTC product\n",
+    "\n",
+    "Water extent mapping requires Radiometrically Terrian Corrected (RTC) Sentinel-1 scenes in order to accurately connect backscatter features to the input HAND data. The next few cells will use HyP3's Python SDK to request an RTC product from HyP3 for the Sentinel-1 granule (scene):\n",
+    "\n",
+    "[S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B](https://search.asf.alaska.edu/#/?zoom=6.964&center=-90.509,40.296&resultsLoaded=true&granule=S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B-METADATA_GRD_HD&searchType=List%20Search&searchList=S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B)\n",
```

Comment:
This URL spins forever trying to load search results, I dropped a message in Data Search Feedback to see if that's a Vertex issue.